docs/sources/installation/centos.md

```diff
@@ -4,48 +4,33 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
 
 # CentOS
 
-While the Docker package is provided by default as part of CentOS-7,
-it is provided by the EPEL repository for CentOS-6. Please note that
-this changes the installation instructions slightly between versions. If you
-need the latest version, you can always use the latest binary which works on
-kernel 3.8 and above.
+Docker is supported on the following versions of CentOS:
 
-These instructions work for CentOS 6 and later. They will likely work for
-other binary compatible EL6 distributions such as Scientific Linux, but
-they haven't been tested.
+- [*CentOS 7 (64-bit)*](#installing-docker---centos-7)
+- [*CentOS 6.5 (64-bit)*](#installing-docker---centos-6.5)
+
+These instructions are likely work for other binary compatible EL6/EL7 distributions
+such as Scientific Linux, but they haven't been tested.
 
 Please note that due to the current Docker limitations, Docker is able to
 run only on the **64 bit** architecture.
 
-To run Docker, you will need [CentOS6](http://www.centos.org) or higher,
-with a kernel version 2.6.32-431 or higher as this has specific kernel
-fixes to allow Docker to run.
+## Kernel support
+
+Currently the CentOS project will only support Docker when running on kernels
+shipped by the distribution. There are kernel changes which will cause issues
+if one decides to step outside that box and run non-distribution kernel packages.
+
+To run Docker on [CentOS-6.5](http://www.centos.org) you will need kernel version
```

Comment:
if you pin a version, like "6.5" here, you'll be needing update regularly. Something like "centos6 (see site for latest version)" or similar, should allow the docs to stay a bit more timeless